core/shared/src/main/scala/org/threeten/bp/Duration.scala

```diff
@@ -1127,6 +1215,21 @@ final class Duration private (private val seconds: Long, private val nanos: Int)
     Math.addExact(result, nanos.toLong)
   }
 
+  /**
+   * Get the nanoseconds part within seconds of the duration.
+   *
+   * The length of the duration is stored using two fields - seconds and nanoseconds. The
+   * nanoseconds part is a value from 0 to 999,999,999 that is an adjustment to the length in
+   * seconds. The total duration is defined by calling getNano() and getSeconds().
+   *
+   * This instance is immutable and unaffected by this method call.
+   *
+   * @return
+   *   the nanoseconds within the second part of the length of the duration, from 0 to 999,999,999
+   * @since 9
+   */
+  def toNanosPart: Int = (nanos / Duration.NANOS_PER_SECOND).toInt
```

Comment:
Sorry, I still don't understand the calculation here. According to the constructor, `nanos` should always be between 0 and 999,999,999. So why do we need to do any calculation here?
https://github.com/cquiroz/scala-java-time/blob/bee7f13e601ee65dec94a426ca5fe2991fa5954c/core/shared/src/main/scala/org/threeten/bp/Duration.scala#L495-L501

Reply:
Right, let's delete `toNanosPart` and release, If somebody needs it PRs are welcome

Reply:
Hm, I don't think it needed to be deleted. I think the correct implementation is to just return `nanos` here :)